File: web_benchmark/main/get/get_normal/Dockerfile.php
FROM php:8.2-cli-alpine

WORKDIR /app

RUN apk add --no-cache git autoconf g++ make mysql-client

RUN pecl install swoole pdo_mysql && \
    docker-php-ext-enable swoole pdo_mysql

COPY server.php .

EXPOSE 8080
CMD ["php", "server.php"]

File: web_benchmark/main/get/get_normal/Dockerfile.go
FROM golang:1.24-alpine

WORKDIR /app

RUN apk add --no-cache git

COPY server.go .
RUN go mod init benchmark && \
    go get github.com/gofiber/fiber/v2 && \
    go get github.com/go-sql-driver/mysql

RUN go build -o server server.go

EXPOSE 8004
CMD ["./server"]
